doc/man3/OSSL_PARAM_get_int.pod

```diff
@@ -0,0 +1,173 @@
+=pod
+
+=head1 NAME
+
+OSSL_PARAM_get_int8, OSSL_PARAM_get_uint8, OSSL_PARAM_get_int16,
```

Comment:
OMG.  Gee that is a lot of functions.  Dozens of them.  Just do int/long and double.